lib/sidekiq/api.rb

```diff
@@ -64,61 +64,65 @@ def queues
     end
 
     def fetch_stats!
-      pipe1_res = Sidekiq.redis do |conn|
+      pipe1_res = Sidekiq.redis { |conn|
         conn.pipelined do
-          conn.get('stat:processed')
-          conn.get('stat:failed')
-          conn.zcard('schedule')
-          conn.zcard('retry')
-          conn.zcard('dead')
-          conn.scard('processes')
-          conn.lrange('queue:default', -1, -1)
+          conn.get("stat:processed")
+          conn.get("stat:failed")
+          conn.zcard("schedule")
+          conn.zcard("retry")
+          conn.zcard("dead")
+          conn.scard("processes")
+          conn.lrange("queue:default", -1, -1)
         end
-      end
+      }
 
-      processes = Sidekiq.redis do |conn|
-        sscan(conn, 'processes')
-      end
+      processes = Sidekiq.redis { |conn|
+        sscan(conn, "processes")
+      }
 
-      queues = Sidekiq.redis do |conn|
-        sscan(conn, 'queues')
-      end
+      queues = Sidekiq.redis { |conn|
+        sscan(conn, "queues")
+      }
 
-      pipe2_res = Sidekiq.redis do |conn|
+      pipe2_res = Sidekiq.redis { |conn|
         conn.pipelined do
-          processes.each {|key| conn.hget(key, 'busy') }
+          processes.each {|key| conn.hget(key, "busy") }
           queues.each {|queue| conn.llen("queue:#{queue}") }
         end
-      end
+      }
 
       s = processes.size
       workers_size = pipe2_res[0...s].map(&:to_i).inject(0, &:+)
       enqueued     = pipe2_res[s..-1].map(&:to_i).inject(0, &:+)
 
       default_queue_latency = if (entry = pipe1_res[6].first)
-                                job = Sidekiq.load_json(entry) rescue {}
-                                now = Time.now.to_f
-                                thence = job['enqueued_at'] || now
-                                now - thence
-                              else
-                                0
-                              end
+        job = begin
```

Comment:
Issue with the linter. 

**TODO:** 
- @gearnode open issue on the linter
- @gearnode write this code with an other way